libxrdp/xrdp_sec.c

```diff
@@ -2133,7 +2133,15 @@ xrdp_sec_process_mcs_data_CS_CORE(struct xrdp_sec *self, struct stream *s)
     {
         client_info->bpp = 32;
     }
-
+    if (earlyCapabilityFlags & 0x100) /* RNS_UD_CS_SUPPORT_DYNVC_GFX_PROTOCOL */
+    {
+        LOG_DEVEL(LOG_LEVEL_INFO, "client supports gfx");
```

Comment:
I think this should be LOG instead of LOG_DEVEL.

Reply:
Fixed. Do you think the "client DOES NOT support gfx" message should also be a LOG?

Reply:
Either is fine. Let's leave it as-is now.

Reply:
So... Should I merge this then?